bin/npm-dependency-fallback.js

```diff
@@ -45,12 +63,34 @@ for (const dependencyName of Object.keys(packageJson.dependencies)) {
         }
     }
 
-    missingDependencies.push(dependency);
+    if (!isYarn) {
```

Comment:
`!isYarn` does not necessarily mean that it is npm, what about storing the package manager name as a string like `'NPM'` and `'YARN'` ?
This would also allow to use a switch where we have code that do "if yarn then A else B" => "case yarn: A; case npm: B;"